README.md

```diff
@@ -38,7 +40,120 @@ class GuiceModule extends AbstractModule {
 By using Gluon Ignite, you not only get dependency injection inside your application class, but also 
 within your FXML controllers too. Even though the sample above shows a Guice context, as mentioned 
 Gluon Ignite also supports other DI frameworks. By using a DaggerContext or SpringContext in your application, 
-it can be easily set up to work with with those frameworks instead.
+it can be easily set up to work with those frameworks instead.
+
+### Micronaut support
+
+Ignite implementation for Micronaut differs from the other implementations, and has much deeper integration with Micronaut framework.
+
+#### _Application main class_
+
+Ignite Micronaut provides a special implementation of JavaFX `Application` class, 
+which has to be configured as your main class: `com.gluonhq.ignite.micronaut.FXApplication`. 
+As an application developer, you just have to implement an entry point into your application, 
+which will be picked up by Micronaut automatically, since it is a normal bean class. 
+
+Here is a simple example:
+```java
+import io.micronaut.context.event.ApplicationEventPublisher;
+import io.micronaut.runtime.event.annotation.EventListener;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import static com.gluonhq.micronaut.FXApplication.StartEvent;
+import static com.gluonhq.micronaut.FXApplication.StopEvent;
+
+@Singleton
+public class AppEntryPoint {
+
+    @Inject
+    private Object something;
+
+    @EventListener
+    void onAppStart(StartEvent event) {
+        Stage stage = event.getStage();
+        Scene scene = new Scene( new Label("Hello, world!"));
+        stage.setScene(scene);
+        stage.setTitle("My application");
+        stage.show();
+    }
+    
+    @EventListener
+    void onAppStop(StopEvent event) {
+        //...
+    }
+
+}
+```
+#### _JavaFX Application Thread_ 
+Any bean method can be forced to run on JavaFX Application Thread using `OnFxThread` annotation
+```java
+@OnFXThread
+void refreshTime() {
+    label.setText(timePattern.format(ZonedDateTime.now()));
+}
+```
+#### _FXML Views_ 
+Ignite Micronaut implements support for FXML views based on naming conventions:
+- A simple empty bean inherited from `FXMLView` defines the view, its name should be exactly 
+  the same as corresponding `fxml` file and should be in the corresponding package. This name usually ends with `View` but does not have to.
```

Comment:
Is the following statement really true?

> This name usually ends with `View` but does not have to.

When I look at the code of `FXMLRootProvider.getViewName()` it seems that it always appends `View` at the end of the class name (which is deduced from the Controller class).

Reply:
I intended for it to be this way :)  Modified version is less strict